web/ui/react-app/src/pages/alerts/CollapsibleAlertPanel.tsx

```diff
@@ -36,14 +36,31 @@ const CollapsibleAlertPanel: FC<CollapsibleAlertPanelProps> = ({ rule, showAnnot
             <div>
               expr: <Link to={createExpressionLink(rule.query)}>{rule.query}</Link>
             </div>
-            <div>
-              <div>labels:</div>
-              <div className="ml-5">severity: {rule.labels.severity}</div>
-            </div>
-            <div>
-              <div>annotations:</div>
-              <div className="ml-5">summary: {rule.annotations.summary}</div>
-            </div>
+            {rule.duration > 0 && (
+              <div>
+                <div>for: {formatRange(rule.duration)}</div>
+              </div>
+            )}
+            {rule.labels && Object.entries(rule.labels).length > 0 && (
+              <div>
+                <div>labels:</div>
+                {Object.entries(rule.labels).map(([key, value]) => (
+                  <div className="ml-4" key={key}>
+                    {key}: {value}
```

Comment:
Although we are not outputting copyable YAML anyway, we should probably at least quote (and quote-escape) the label values, since they can contain arbitrary Unicode characters.

Reply:
Yeah, as it's an existing bug anyway, feel free to do it later.